package.json

```diff
@@ -5,16 +5,15 @@
   "main": "index.js",
   "scripts": {
     "build": "tsc --build",
-    "build:all": "npm run build:workspaces && npm run build && npm run build-declaration && npm run build:docs",
+    "build:all": "npm run build:workspaces && npm run build && npm run build:docs",
     "build:docs": "typedoc",
     "build:workspaces": "npm run build --workspaces --if-present",
-    "build-declaration": "tsc --declaration",
     "watch": "tsc --build -w",
     "prettier": "prettier -w src/",
     "lint": "eslint --fix",
     "format": "npm run prettier && npm run lint",
     "ci-lint": "eslint && prettier --check src/",
-    "prepublishOnly": "npm run lint && npm run build:all && npm run build-declaration",
+    "prepublishOnly": "npm run lint && npm run build:all",
```

Comment:
I thought I removed these elsewhere, but can't remember now. A separate `tsc --declaration` is not necessary now that `declaration: true` is in the `tsconfig.base.json`